gatherer/src/main/java/de/zalando/pgobserver/gatherer/StatStatementsGatherer.java

```diff
@@ -114,11 +133,12 @@ protected boolean gatherData() {
         }
     }
 
-    public String getQuery() {
+    private String getQuery() {
 //        and not (usename = 'postgres' and upper(query) like 'COPY %') ?
         String sql = "with q_data as (\n" +
                 "      select\n" +
                 "      now() as timestamp,\n" +
+                "      s.userId,\n" +
```

Comment:
I am agree about username and i will change the implementation to persist username instead of userid and its more meaningful.